pkg/controlplane/reconcilers/endpointsadapter_test.go

```diff
@@ -232,7 +232,7 @@ func TestEndpointsAdapterUpdate(t *testing.T) {
 	// with one that has an IPv4 address type.
 	endpoints4, _ := generateEndpointsAndSlice("foo", "testing", []int{80}, []string{"10.1.2.7", "10.1.2.8"})
 	_, epSlice4IP := generateEndpointsAndSlice("foo", "testing", []int{80}, []string{"10.1.2.7", "10.1.2.8"})
-	epSlice4IP.AddressType = discovery.AddressTypeIP
+	epSlice4IP.AddressType = discovery.AddressType("IP")
```

Comment:
this could use a comment on why an "invalid" type is being used

Reply:
Updated, thanks!